miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -179,7 +190,8 @@ abstract class MiniApp internal constructor() {
                     initVerifier = { CachedMiniAppVerifier(context) }
                 ),
                 miniAppInfoFetcher = MiniAppInfoFetcher(apiClient),
-                miniAppCustomPermissionCache = MiniAppCustomPermissionCache(context)
+                miniAppCustomPermissionCache = customPermissionCache,
+                miniAppManifestCache = MiniAppManifestCache(context, customPermissionCache)
```

Comment:
You don't need to init them here, you can init them in with `lazy` in `RealMiniApp`.